docs/how-to-guides/themes/README.md

```diff
@@ -1,6 +1,20 @@
 # Themes
 
-The block editor provides a number of options for theme designers and developers, including theme-defined color settings, font size control, and much more.
+The block editor provides a number of options for theme designers and developers, to interact with it, including theme-defined color settings, font size control, and much more.
+
+## Types of themes
+
+### Classic theme
+
+In terms of block editor terminology this is any theme that defines its templates in the traditional `.php` file format, and that doesn't have an `index.html` format template in the `/block-templates` or `/templates` folders. A `Classic` theme still has the ability to provide configuration and styling options to the block editor, and block content, via [Theme Supports](/docs/how-to-guides/themes/theme-support.md). 
+
+### Hybrid theme
```

Comment:
@annezazu, @youknowriad  - have we got this terminology correct in terms of a Classic theme that has a theme.json file? We thought `Hybrid` was a term that was being used, but weren't 100% sure of that.

Reply:
The other approach is to not name this combination, and just note under Classic theme that they can provide config via either Theme Supports or a theme.json

Reply:
The definition of a hybrid theme is different that what is mentioned here. A hybrid theme is a theme that has both "php templates" and "block templates", meaning depending on the WP version can work as a block theme or classic theme (block theme in newer versions).

I think though that it's probably not worth mentioning in the docs. I don't expect it to be so common.